app/views/partials/case_heading.scala.html

```diff
@@ -22,12 +22,12 @@
 @(c: Case, displayBackLink: Boolean = true)(implicit messages: Messages, request: Request[_])
 
 <div id="case-heading" class="case-heading mt-1 mb-2">
-
+    <br>
```

Comment:
Could we just increase the bottom margin instead of adding a line break?

Reply:
Of course, we have changed it! Thanks! 👍